designsafe/apps/workspace/templates/designsafe/apps/workspace/app_variant_plugin_button.html

```diff
@@ -1,6 +1,10 @@
 {# @var variant #}
 
-        {% if variant.enabled %}
+        {% if variant.download_href %}
```

Comment:
This will show even if the variant is disabled. Do we want that behavior?

Reply:
Nope. Thanks. Fixed _I think_ by d385df4.